app/src/main/java/net/gsantner/markor/format/TextFormat.java

```diff
@@ -104,6 +106,7 @@ public static TextFormat getFormat(int formatId, Activity activity, Document doc
     private TextActions _textActions;
     private Highlighter _highlighter;
     private TextConverter _converter;
+    private TextWatcher _watcher = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/gsantner/markor/pullRequest?prid=5382883)